src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.util;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String CREDENTIALS_PATH =
+            "src/main/resources/credentials.env";
+
+    public static Connection getConnection() {
+        try (BufferedReader reader = new BufferedReader(new FileReader(CREDENTIALS_PATH))) {
+            String className = "com.mysql.cj.jdbc.Driver";
+            Class.forName(className);
+            String[] credentials = new String[2];
+            credentials[0] = reader.readLine();
```

Comment:
Ensure that the `credentials.env` file contains exactly two lines: the username on the first line and the password on the second line. If the file format changes, this code will not work correctly.